chains/mainnet/noble.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  chainID: 'noble-1',
+  lcd: 'https://noble-api.polkachu.com',
+  gasAdjustment: 1.75,
+  gasPrices: { uosmo: 0.025 }, // <-- update this
```

Comment:
Right now gas prices are 0 ibc/Atom ("ibc/EF48E6B1A1A19F47ECAEA62F5670C37C0580E86A9E88498B7E393EB6F49F33C0")

and 0 uusdc

However, this will change after uusdc is officially launched.